ui/src/app/views/profile-page/component.html

```diff
@@ -61,3 +61,18 @@ <h3>Identity Verification</h3>
     </div>
   </ng-template>
 </div>
+<div>
+  <button class="btn" (click)="submitTermsOfService()" [disabled]="email===null">
```

Comment:
Can we have some status indicator which shows if you have already done these things?